src/main/java/seedu/tracker/model/module/Mc.java

```diff
@@ -11,8 +11,11 @@
  */
 public class Mc {
 
-
-    public static final String MESSAGE_CONSTRAINTS = "MC credit should only be an Integer from 1-999.";
+    public static final String OVERALL_MESSAGE_CONSTRAINTS = "MC credit for a module should only be an "
+            + "Integer from 1-20.\nMC credit for a MC goal should only be an Integer from 1-999";
```

Comment:
Would it be better to say "MC" instead of "MC credit"? The user should know that MC represents Modular Credits